src/features/MentorPage/MentorCard/MentorCardHeaderBasicInfo.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
+import { Text } from '../../../components/CommonTextStyles/Text';
+import CSS from 'csstype';
+import MentorCardLanguages from './MentorCardLanguages';
+
+type Props = {
+  mentorCardData: ListCardProps;
+};
+
+const customStylus: CSS.Properties = {
+  margin: '0px',
+  display: 'flex',
+  flexWrap: 'wrap',
+};
+
+const truncateText: CSS.Properties = {
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  margin: '0px',
+  width: '100%',
+  fontFamily: '"Source Sans Pro"',
+  fontStyle: 'normal',
+  fontWeight: '400',
+  fontSize: '1rem',
+  lineHeight: '1.5rem',
+  color: '#ffffff',
+};
+
+const MentorCardHeaderBasicInfo = ({ mentorCardData }: Props) => {
+  const age = new Date().getFullYear() - mentorCardData.mentor.birthYear;
+  return (
+    <BasicInfoContainer>
+      <Text color="white" variant="h2" style={{ margin: 0 }}>
+        {mentorCardData.mentor.displayName}
+      </Text>
+      <NameDivider />
+      <Text color="white" variant="p" style={customStylus}>
+        {age} v. <StyledDivider>|</StyledDivider> {mentorCardData.mentor.region}
+      </Text>
+      <Text color="white" variant="p" style={{ margin: 0 }}>
+        {mentorCardData.isLoggedIn}
+      </Text>
```

Comment:
Is this supposed to show something when mentor is logged in?
Btw, currently we cannot know if someone is logged in, but I guess these are from the designs.

Reply:
There will be a "ei tavoitettavissa" text if not logged in, but actually I'm not sure if it is about being logged in

Reply:
Yeah, it is probably the "vacation"-mode we are talking about. 
Mentor can set themselves on vacation, and then actors cannot start a new conversation with them (only existing ones can continue messaging)

But in any case, this boolean isLoggedIn does not evaluate to a string - it is an empty <p> there

Reply:
I'm changing this to the functionality branch!